src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -10764,6 +10894,54 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/ActionDefinition'
+  RulesetPaging:
+    type: object
+    required:
+      - list
+    properties:
+      list:
+        type: object
+        required:
+          - pagination
+          - entries
+        properties:
+          pagination:
+            $ref: '#/definitions/Pagination'
+          entries:
+            type: array
+            items:
+              $ref: '#/definitions/RulesetEntry'
+  RulesetEntry:
+    type: object
+    required:
+      - entry
+    properties:
+      entry:
+        $ref: '#/definitions/Ruleset'
+  Ruleset:
+    type: object
+    required:
+      - id
+    properties:
+      id:
+        description: Identifier for the ruleset
+        type: string
+  LinkedRulesetBodyCreate:
+    type: object
+    required:
+      - id
+      - type
+    properties:
+      id:
+        description: Identifier for either the folder or the ruleset
+        type: string
+      type:
+        description: Whether id refers to a folder or a ruleset
+        type: string
+        default: RULESET
+        enum:
+          - RULESET
+          - FOLDER
```

Comment:
How/when do we use one or the other ?

Reply:
`{
  "id": "example-ruleset-id"
  "type": "RULESET"
}`
or
`{
  "id": "example-node-id"
  "type": "FOLDER"
}`
Similar to what we have for creating group memberships: https://apadev.envalfresco.com/api-explorer/#/groups/createGroupMembership

Reply:
In the case of "FOLDER", we could assume it's similar to `-default-` in the path params of the GET or DELETE, as in it links to the only ruleset of the folder ID provided. (This being an issue only if we do eventually implement multiple rulesets per folder). Do you think that is a reasonable approach? The alternative is requiring the user link to specifically to a ruleset id and never a folder id